integration-tests/cli/csv-parser-price.it.js

```diff
@@ -22,123 +22,105 @@ describe('CSV and CLI Tests', () => {
-    it('should print usage information given the help flag', done => {
-      exec(`${binPath} --help`, (error, stdout, stderr) => {
-        expect(String(stdout)).toMatch(/help/)
-        expect(error && stderr).toBeFalsy()
-        done()
-      })
+    it('should print usage information given the help flag', async () => {
+      const [stdout, stderr] = await exec(`${binPath} --help`)
+      expect(stderr).toBeFalsy()
+      expect(stdout).toMatchSnapshot()
     })
 
-    it('should print the module version given the version flag', done => {
-      exec(`${binPath} --version`, (error, stdout, stderr) => {
-        expect(stdout).toBe(`${version}\n`)
-        expect(error && stderr).toBeFalsy()
-        done()
-      })
+    it('should print the module version given the version flag', async () => {
+      const [stdout, stderr] = await exec(`${binPath} --version`)
+      expect(stderr).toBeFalsy()
+      expect(stdout).toBe(`${version}\n`)
     })
 
-    it('should write output to file', done => {
+    it('should write output to file', async () => {
       const csvFilePath = path.join(samplesFolder, 'simple-sample.csv')
       const jsonFilePath = tmp.fileSync().name
 
-      // eslint-disable-next-line max-len
-      exec(
-        `${binPath} -p ${projectKey} -i ${csvFilePath} -o ${jsonFilePath}`,
-        (cliError, stdout, stderr) => {
-          expect(cliError && stderr).toBeFalsy()
-
-          fs.readFile(jsonFilePath, { encoding: 'utf8' }, (error, data) => {
-            expect(data.match(/prices/)).toBeTruthy()
-            expect(error).toBeFalsy()
-            done()
-          })
-        }
+      await exec(
+        `${binPath} -p ${projectKey} -i ${csvFilePath} -o ${jsonFilePath}`
       )
+
+      const data = await fs.readFile(jsonFilePath, { encoding: 'utf8' })
+      expect(data).toMatchSnapshot()
     })
   })
 
   describe('CLI logs specific errors', () => {
-    it('on faulty CSV format', done => {
+    it('on faulty CSV format', async () => {
       const csvFilePath = path.join(samplesFolder, 'faulty-sample.csv')
       const jsonFilePath = tmp.fileSync().name
 
-      exec(
-        `${binPath} -p ${projectKey} -i ${csvFilePath} -o ${jsonFilePath}`,
-        (error, stdout, stderr) => {
-          expect(error.code).toBe(1)
-          expect(stdout).toBeFalsy()
-          expect(stderr.match(/Row length does not match headers/)).toBeTruthy()
-          done()
-        }
-      )
+      try {
+        await exec(
+          `${binPath} -p ${projectKey} -i ${csvFilePath} -o ${jsonFilePath}`
+        )
+      } catch (error) {
+        expect(error.code).toBe(1)
+        expect(String(error)).toMatch(/Row length does not match headers/)
+      }
     })
 
-    it('on parsing errors', done => {
-      const csvFilePath = path.join(samplesFolder, 'missing-type-sample.csv')
-      const jsonFilePath = tmp.fileSync().name
+    it(
+      'on parsing errors',
+      async () => {
```

Comment:
I think this can all be on the same line:
```js
it('on parsing errors', async () => {
```

Reply:
Prettier..